cms/datasets/blocks.py

```diff
@@ -29,6 +32,53 @@ class Meta:
         template = "templates/components/streamfield/dataset_link_block.html"
 
 
+class TimeSeriesPageLinkBlock(StructBlock):
+    title = CharBlock(required=True)
+    description = TextBlock(required=True)
+    url = URLBlock(required=True)
+
+    class Meta:
+        icon = "link"
+        template = "templates/components/streamfield/time_series_link.html"
+
+    def clean(self, value: "StructValue") -> "StructValue":
+        """Checks that the given time series page URL matches the allowed domain."""
+        errors = {}
+
+        if not value["url"].startswith(settings.ONS_WEBSITE_BASE_URL):
```

Comment:
Currently, we throw a validation error if the URL doesn't begin with `https://www.ons.gov.uk`, is there a specific reason for this? Can't we also allow `www.ons.gov.uk`? As it would also be valid, and when I visit the URL `www.ons.gov.uk`, it all works fine and uses `https` by default.